src/Pages/Home.tsx

```diff
@@ -0,0 +1 @@
+export const Home = () => <h1 className="title">Home page</h1>;
```

Comment:
For example, here we can use this div, to wrap our page. 
It is not correct when you render the page and return only the title

```suggestion
export const Home = () => (
  <div className="container">
    <h1 className="title">Home page</h1>
  </div>
);
```